client/src/pages/UploadReceipt.js

```diff
@@ -1,37 +1,127 @@
+    margin: '50px 0'
+  }
 });
 
 class UploadReceipt extends Component {
   state = {
-    image: ''
+    images: null,
+    selectedImages: null
   }
 
   onFormChange = (e) => {
-    this.setState({ [e.target.name]: e.target.files[0] });
+    this.setState({ [e.target.name]: e.target.files, selectedImages: URL.createObjectURL(e.target.files[0]) });
   }
 
   onBtnClick = () => {
+    let imagesList = new FormData();
+    for (let i = 0; i < this.state.images.length; i++) {
+      imagesList.append('files', this.state.images[i]);
+    }
     console.log(this.state);
-    this.props.setPage("create");
+    /*
+    fetch("/receipts/images", {
+      method: "POST",
+      // headers: { "Content-Type": "multipart/form-data" },
+      body: imagesList
+    })
+      .then(res => {
+        return res.json();
+      })
+      .then(data => {
+        console.log(data);
+        this.props.setImgUrls(data['locations']);
+        this.props.setPage("create");
+      })
+      .catch(err => {
+        console.log(err);
+      })
+    */
   };
 
   render() {
     const { classes } = this.props;
     return (
       <div className={classes.root}>
-        <Typography variant="h4">Upload Receipt</Typography>
-        <Divider />
-        <FormControl>
-          <input type="file" name="image" accept="image/*" onChange={this.onFormChange} />
-        </FormControl><br />
-        <Button className={classes.button} onClick={this.onBtnClick} >Submit</Button>
+        <Grid container spacing={3}>
+          <Grid item xs={6}>
+            <div className={classes.paper}>
+              <Typography style={{color: '#1A3360', margin: 30, fontWeight: 'bold'}} variant="h4">Upload receipt</Typography>
+              <Typography style={{color: 'gray', marginLeft: 30}} variant="body1">Upload one or more receipt(s)</Typography>
+              <CardMedia image={this.state.selectedImages} title="receipt image" className={classes.image} />
+              {this.state.selectedImages && <DeleteIcon />}
+              <Button variant="outlined" className={classes.submit} onClick={this.onBtnClick} >Submit</Button>
+            </div>
+          </Grid>
+          <Grid item xs={6}>
+            <div className={classes.paper}>
+              <FormControl>
+                <label htmlFor="raised-button-file">
+                  <div 
+                    className={classes.drop}
+                    onDrop={(e) => {
+                      e.preventDefault();
+                      console.log(e);
+                    }}
+                    onDragOver={(e) => e.preventDefault()}
+                  >
+                    <CardMedia style={{marginLeft: 'auto', marginRight: 'auto'}} image={DragDropIcon} title="Drag and Drop Icon" className={classes.image} /><br />
+                    <Typography style={{margin: 12}} variant="h5" align="center">Drop files here</Typography>
+                    <Typography style={{margin: 12}} variant="body1" align="center">or</Typography>
```

Comment:
may be good to create a class for these 2 Typography cause they both have `margin: 12` and `align: center` if it's to center text, you can actually just assign this to the parent elem too!